packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -494,6 +495,11 @@ export class DashboardModel {
                 `${DashboardTilesTableName}.height`,
                 `${DashboardTilesTableName}.dashboard_tile_uuid`,
                 `${SavedChartsTableName}.saved_query_uuid`,
+                this.database.raw(`CASE
+                     WHEN ${SavedChartsTableName}.dashboard_uuid IS NOT NULL
+                        THEN TRUE
+                        ELSE FALSE
+                     END AS belongs_to_dashboard`),
```

Comment:
If you want it shorter you can do this I think
```suggestion
               this.database.raw(`${SavedChartsTableName}.dashboard_uuid IS NOT NULL AS belongs_to_dashboard`),

```